docs/ai/integrations/langchain.md

```diff
@@ -9,44 +9,34 @@ Integrate Unity Catalog AI with [LangChain](https://python.langchain.com) to sea
 Install the Unity Catalog AI LangChain integration from PyPI:
 
 ```sh
-pip install unitycatalog-langchain
+pip install unitycatalog-langchain langchain
```

Comment:
unitycatalog-langchain includes langchain as dependency, we don't need to install langchain explicitly :)

Reply:
And it's true for all flavors we support :D

Reply:
ohhhhh yeah lol. I'll update ;)